extensions/podman/packages/extension/scripts/podman-download.ts

```diff
@@ -138,17 +119,16 @@ export class PodmanDownload {
     return this.#shaCheck;
   }
 
-  protected getDownloadAndCheck(): DownloadAndCheck {
-    return this.#downloadAndCheck;
+  protected getArtifactsToDownload(): {
+    version: string;
+    downloadName: string;
+    artifactName: string;
+  }[] {
+    return this.#artifactsToDownload;
   }
 
   async downloadBinaries(): Promise<void> {
-    // fetch from GitHub releases
```

Comment:
I think this bloc should not be removed

Reply:
I am not speaking about the machine images but rather to the Podman installers that are not bundled anymore with this removal